### PR TITLE
Progress to msg

### PR DESCRIPTION
**Pull request by macMikey**
Give some live feedback to the developer during the 3-10 sec that the application is being created.

### CODE REVIEW

**Comment by macMikey** [comment]
Issue #66

File: utils/create_levure_app_files/stack.livecodescript [hunk 1]
```diff
@@ -524,7 +539,7 @@ on fileSetPermissions pFile, pOwner, pGroup, pAll
   put _escapeForShell(pFile) into pFile
 
   switch the platform
-    case "Win32"
+  case "Win32"
```
Comment (trevordevore) [inline utils/create_levure_app_files/stack.livecodescript]: @macMikey These white space changes shouldn't be part of this PR. Will you please remove them?

**Comment by macMikey** [comment]
Hmm.  I wonder why the whitespace changed.
The changes in the script file were just applying the changes to the .livecodescript file to the stack script, because you don't have the .livecodescript file set as a behavior for the stack.  It seems that the SOS was copied/pasted to the stack script.  I forgot to ask you if you would like me to remove the stack script and assign the behavior.

**Comment by macMikey** [comment]
Try that - but first let me know if you want to remove the stack script and replace it with the SOS as a behavior.

**Comment by trevordevore** [comment]
@macMikey If you look at the diff the white space changes look like the script editor changing indentation.

As for the stack - I remember why it is designed that way. When the stack is downloaded through the message box for use in creating a Levure app you can't have an SOS behavior. If you look at the card script of the stack there is a script that updates custom props and stack script with the latest and greatest versions.

So the changes that need to be made to this PR are as follows:

1. Undo the whitespace changes so that the diff for this PR only shows the actual code changes you made.
2. Open the .livecode stack and save it. That will suck up the latest YAML files and your script changes. Commit that version of the binary stack.

**Comment by macMikey** [comment]
I undid the whitespace (yes, probably SE auto-thang), and re-pushed.  Have a look at that.  In the meantime I'll reload/re-save the stack and push that.

**Comment by trevordevore** [comment]
@macMikey Whitespace issue is resolved. Let me know once you've pushed the updated binary.

**Comment by macMikey** [comment]
I did.  I just checked the commit.  It's listed above the .livecodescript file.

**Comment by macMikey** [comment]
What else is open on this?

**Comment by trevordevore** [comment]
@macMikey Nothing. I just have to review it and test it to ensure it works as advertised. I'm making a couple of tweaks to the logger docs changes. You added "error" as supported type in the docs but it isn't really a supported type.

**Comment by macMikey** [comment]
WHOOPS!  "error" is a mistake.  I'm working on adding "error", but I'm not done with it yet.  I was trying to have it done in October so I can get my really, really important t-shirt from doing 5 levure PR's.
